BUILD.gn

```diff
@@ -102,6 +102,10 @@ group("libopenthread-spinel-rcp") {
 
 if (current_os == "fuchsia") {
   group("lib-ot-core") {
-    public_deps = [ ":libopenthread-ftd" ]
+    public_deps = [
```

Comment:
nit: Run "gn format" on this file (and the others).

Reply:
Thank you Michael, 
I ran "gn format" on all the files.